features/features.go

```diff
@@ -39,6 +40,7 @@ var features = map[FeatureFlag]bool{
 	WildcardDomains:         false,
 	EnforceChallengeDisable: false, // deprecated
 	TLSSNIRevalidation:      false,
+	DontCancelCTSubmissions: false,
```

Comment:
Let's change the name and default sense of this flag: "CancelCTSubmissions: true"